src/runtime/GlobalObjectBuiltinRegExp.cpp

```diff
@@ -27,6 +27,28 @@
 
 namespace Escargot {
 
+#ifdef ESCARGOT_ENABLE_ES2015
+static size_t advanceStringIndex(String* str, size_t index, bool unicode)
```

Comment:
`advanceStringIndex` was also added by #211 , I think it would be better to use that.

Reply:
Most of this was implemented before #211 landed, i'll change this to use that then.